packages/query/src/queriers/shielded-pool.ts

```diff
@@ -14,7 +14,7 @@ export class ShieldedPoolQuerier implements ShieldedPoolQuerierInterface {
     this.client = createClient(grpcEndpoint, QueryService);
   }
 
-  async assetMetadata(assetId: AssetId): Promise<Metadata | undefined> {
+  async assetMetadataById(assetId: AssetId): Promise<Metadata | undefined> {
```

Comment:
Not strictly necessary for this PR, but I figured it made sense to align the method name with the RPC method.